### PR TITLE
Migrate to ghcr

### PR DESCRIPTION
**Pull request by tei-k**
# Description

https://scalar-labs.atlassian.net/browse/DLT-7849

### CODE REVIEW

File: docs/installation-with-docker.md [hunk 1]
```diff
@@ -28,13 +28,12 @@ $ cd scalar-samples
 ## Docker login
 
 `docker login` is required to start the Scalar DL Docker image. Because the
-[`scalar-server`](https://hub.docker.com/r/scalarlabs/scalar-server/) repository
-on Docker Hub is currently private, your Docker account needs to be listed as a
-collaborator of the repository. Ask a person in charge to get your account
-ready.
+[`scalar-server`](https://github.com/orgs/scalar-labs/packages/) repository
```
Comment (feeblefakie) [inline docs/installation-with-docker.md:31]:
```suggestion
[`scalar-ledger`](https://github.com/orgs/scalar-labs/packages/container/package/scalar-ledger) repository
```

File: docs/installation-with-docker.md [hunk 2]
```diff
@@ -28,13 +28,12 @@ $ cd scalar-samples
...
+on GitHub Container Registry is currently private, your GitHub account needs to be set with permissions to access the container images.
+Ask a person in charge to get your account ready.
 
 ```
-$ docker login
+$ docker login ghcr.io
 ```
 
 ## Start up the server
```
Comment (feeblefakie) [inline docs/installation-with-docker.md:39]:
```suggestion
## Start up Scalar DL Ledger
```

Comment (feeblefakie) [reply docs/installation-with-docker.md:39]: Can you also update the sentence below to this?

```The following command starts up Scalar DL Ledger along with the back-end
Cassandra server in the Docker containers. At the first time you run the
command, the required Docker images will be downloaded from Docker Hub.```

**Review by feeblefakie — COMMENTED** [review]
Thank you! Left some suggestions.

File: docs/installation-with-docker.md (hunk 1 above)
Comment (tei-k) [reply docs/installation-with-docker.md]: Fixed in 2b3bf75.

File: docs/installation-with-docker.md (hunk 2 above)
Comment (tei-k) [reply docs/installation-with-docker.md]: Fixed in 5df8523.

Comment (ymorimo) [reply docs/installation-with-docker.md]: https://github.com/scalar-labs/scalardl/pull/9/files#diff-2e9d792c66c60f65e3e5367a87f6145398069017f0aa900529fc19ffaacfd5dcL42-R42

"Docker Hub" should be changed, and the first line should not be removed.

Comment (tei-k) [reply docs/installation-with-docker.md]: Thanks! Fixed in 05ebbda

File: docs/installation-with-docker.md [hunk 3]
```diff
@@ -28,20 +28,17 @@ $ cd scalar-samples
...
+$ docker login ghcr.io
 ```
 
-## Start up the server
+## Start up Scalar DL Ledger
 
-The following command starts up the Scalar DL server along with the back-end
-Cassandra server in the Docker containers. At the first time you run the
-command, the required Docker images will be downloaded from Docker Hub.
+The following command starts up the Scalar DL server along with the back-end Cassandra server in the Docker containers. At the first time you run the command, the required Docker images will be downloaded from GitHub Container Registry.
```
Comment (feeblefakie) [inline docs/installation-with-docker.md:41]:
```suggestion
The following command starts up the Scalar DL ledger along with the back-end Cassandra server in the Docker containers. At the first time you run the command, the required Docker images will be downloaded from GitHub Container Registry.
```

File: docs/installation-with-docker.md [hunk 4]
```diff
@@ -28,20 +28,17 @@ $ cd scalar-samples
...
+$ docker login ghcr.io
 ```
 
-## Start up the server
+## Start up Scalar DL Ledger
 
-The following command starts up the Scalar DL server along with the back-end
-Cassandra server in the Docker containers. At the first time you run the
-command, the required Docker images will be downloaded from Docker Hub.
+The following command starts up the Scalar DL ledger along with the back-end Cassandra server in the Docker containers. At the first time you run the command, the required Docker images will be downloaded from GitHub Container Registry.
```
Comment (feeblefakie) [inline docs/installation-with-docker.md:41]:
```suggestion
The following command starts up Scalar DL Ledger along with the back-end Cassandra server in the Docker containers. At the first time you run the command, the required Docker images will be downloaded from GitHub Container Registry.
```

**Review by feeblefakie — APPROVED** [review]
LGTM! Thank you!

**Pull request merged**